code/modules/clothing/suits/ego_gear/teth.dm

```diff
@@ -91,3 +91,9 @@
 	desc = "Now the straitjacket is nothing but an accessory to the hatred and anger that remains."
 	icon_state = "regret"
 	armor = list(RED_DAMAGE = 20, WHITE_DAMAGE = -20, BLACK_DAMAGE = 20, PALE_DAMAGE = 0)
+
+/obj/item/clothing/suit/armor/ego_gear/cute
+	name = "SO CUTE!!!"
+	desc = "Ppodae is oh so fluffy and soft. It’s so sweet it could cure your depression."
+	icon_state = "cute"
+	armor = list(RED_DAMAGE = 20, WHITE_DAMAGE = -20, BLACK_DAMAGE = 20, PALE_DAMAGE = 0) // 20
```

Comment:
Armor stats is also the same as Regret, maybe try 2/0/0/0 or 2/-2/0/2?

Reply:
Giving pale defense to teth is kinda weird, so I guess I'll just put 2 on red
2/-2/2/0 is the stat from wiki. I think this problem will be very common when we get more abnos in